hack/set-odh-release.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -o errexit
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+set -o errtrace
+set -o pipefail
+
+k=$(find . -name kfdef.yaml)
+
+latest_version=$(basename $(curl -Ls -o /dev/null -w %{url_effective} https://github.com/opendatahub-io/odh-manifests/releases/latest))
+
+for kfdef in $k; do
+    if ! yq e -e ".metadata.name == \"opendatahub\"" $kfdef 2>&1 >/dev/null; then
```

Comment:
A small nitpick here, since this script requires `yq` I'd suggest moving this conditional statement outside of the if statement so it can be caught by errexit.

Reply:
Sorry @HumairAK I'm a bit confused here and I don't probably fully understand what you want me to do. This if is checking on the exit status of the `yq` command - if `yq` exits with `0` we **don't** skip the file (checking for the opposite - `!=` instead of `==` results in ugly error prints)

I'm not that strong in bash (and I actually don't like it at all), so if you know about any better way to check on exit status with the `errexit` flag on, please help me out.

Reply:
hrmm I guess what I'm trying to say is if the user doesn't have `yq` installed, this loop will still run to completion instead of breaking early, and just complain every iteration that `yq` command not found. But I suppose if you want to use the exit status as part of the conditional then let's just go with it. A work around would be to maybe do maybe add something like `yq --version > /dev/null` in the beginning, so it can fail early if yq is not installed.

This isn't that big of a deal, let's just go with it.